+reconstruction
- technical, no reco changes
- note the following warning about CentralityTableProducer (cc @mandrenguyen and HI colleagues) 
```
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_2_X_2021-11-21-0000/src/RecoHI/HiCentralityAlgos/plugins/CentralityTableProducer.cc:39:1: warning: CentralityTableProducer inherits from edm::EDProducer,edm::EDFilter,edm::EDAnalyzer, or edm::OutputModule [deprecated.edmChecker]
```

--- a/RecoHI/HiCentralityAlgos/plugins/CentralityTableProducer.cc
+++ b/RecoHI/HiCentralityAlgos/plugins/CentralityTableProducer.cc
@@ -59,7 +59,6 @@ private:
   string rootTag_;
   ofstream text_;
 
-  CentralityTable* CT;
   const CentralityBins* CB;
 
   unsigned int runnum_;

